src/pages/New.js

```diff
@@ -22,23 +22,29 @@ const AddCircleIcon = styled(Add)`
 
 export default function New() {
   function handleAccept() {
-    postRestaurant({
+    return postRestaurant({
```

Comment:
you don't have to return something in this function.
the `handleAccept` is called when you clicked the accept button in the modal.
the button do not expect a return value from the callback.